File: 1_SECUENCIALES/Secuenciales_3.java
/*
 * Para el calculo del cobro de un préstamo, se pide que para la cuantificación del pago se tome
en cuenta el dinero prestado, el interés anual y los años en los que la deuda se cancela.
 */
import java.util.Scanner;
public class Secuenciales_3 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Prestamo: ");
        int a=sc.nextInt();
        System.out.print("Interes: ");
        int b=sc.nextInt();
        System.out.print("Años: ");
        int c=sc.nextInt();
        float Cobro=a*(1+(b/100)*c);
        System.out.println("Cobro:"+Cobro);
        sc.close();
    }
}

File: 1_SECUENCIALES/Secuenciales_3.py
'''
Para el calculo del cobro de un préstamo, se pide que para la cuantificación del pago se tome
en cuenta el dinero prestado, el interés anual y los años en los que la deuda se cancela.
'''
a=int(input("Prestamo: "))
b=int(input("Interes: "))
c=int(input("Años: "))
Cobro=a*(1+(b/100)*c)
print(f"{Cobro:.2f}")